README.md

```diff
@@ -32,4 +32,9 @@ Old coverage data is stored within `lib/data/countries` for the various habitats
 `lib/data/habitat_coverage_protection/<habitat>_country_output.csv` 
 
 Redlist data for the various species is store in `lib/data/iucn_redlist`. If you need to update just the occurrences of each
-species run `rake import:occurrences`.
\ No newline at end of file
+species run `rake import:occurrences`.
+
+## CSV downloads
+
+- Generating CSV downloads for habitat statistics in each country can be done simply via `rake generate:national_csvs`, which are
```

Comment:
Should this be added to seeds.rb?

Reply:
No, it's not necessary because this is only really important for production.